src/TensorFlowNET.Core/Status/Status.cs

```diff
@@ -60,19 +72,17 @@ public void Check(bool throwException = false)
         {
             if (Code != TF_Code.TF_OK)
             {
-                Console.WriteLine(Message);
+                var message = Message;
+                Console.WriteLine(message);
                 if (throwException)
-                    throw new TensorflowException(Message);
+                    throw new TensorflowException(message);
             }
         }
 
-        public static implicit operator IntPtr(Status status)
-            => status._handle;
```

Comment:
📝 The implicit operators are _extremely_ problematic, and make the type impossible to use reliably. Explicit operators are equally impossible to use correctly, so I'm removing all of these operators as I work my way through the various types of native resources.

Reply:
OK, I don't even know this tricky.